### ISSUE TITLE
Export of tables with Timestamp/Datetime/Time columns defined with ON UPDATE clause with precision fails

### ISSUE THREAD

**Issue by ncwgf** [phpmyadmin/phpmyadmin#12610]
### Steps to reproduce
1. import below SQL to create a table, it's output from `mysqldump`
 - column `id` is **PRIMARY KEY**
 - column `rTime` have attribute `on update` with a `length` 3
2. export this table `aa` in SQL format 
3. You will see it's missed index part in the result
```
DROP TABLE IF EXISTS `aa`;

CREATE TABLE `aa` (
  `id` int(11) NOT NULL,
  `rTime` timestamp(3) NOT NULL DEFAULT '0000-00-00 00:00:00.000' ON UPDATE CURRENT_TIMESTAMP(3),
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
```

### Expected behaviour
Index part should appear on result
```
--
-- Indexes for dumped tables
--

--
-- Indexes for table `aa`
--
ALTER TABLE `aa`
  ADD PRIMARY KEY (`id`);
```

### Actual behaviour
Index part missed, on every related to SQL dump
e.g. Copy table, Copy database, export as SQL format to file or text view

When export/copy database, only the table have the special column `rTime` will not have indexes

If change the column to
- length = 0
- datetime and remove `on update` attribute

indexes part will comeback

### Server configuration
**Host Operating system**:
CentOS Linux release 7.2.1511
below package/software are running in docker-compose environment

**Web server:**
nginx/1.11.4, Official docker image nginx:alpine

**Database:**
10.1.17-MariaDB-1~jessie - mariadb.org binary distribution
 Official docker image mariadb

**PHP version:**
both of
 - 7.0.7 
 - 7.0.11

**phpMyAdmin version:**
both of
 - 4.6.5-dev, *Git revision*: f54f8fa from QA_4_6 branch
 - 4.6.4

### Client configuration
**Browser:**
Google Chrome Version 53.0.2785.143 m (64-bit)
**Operating system:**
Windows 10 Home v1607, build 14393.222

**Comment by devenbansod** [comment]
Should be fixed with phpmyadmin/sql-parser@c38ccad109210c4d504530d01690644c6ca4fa33

**Comment by devenbansod** [comment]
Fixed with updating of SQL parser to 3.4.11 in 1ec99a
